java/impl/aws/src/main/java/co/worklytics/psoxy/S3Handler.java

```diff
@@ -0,0 +1,96 @@
+package co.worklytics.psoxy;
+
+import co.worklytics.psoxy.aws.DaggerAwsContainer;
+import co.worklytics.psoxy.gateway.ConfigService;
+import co.worklytics.psoxy.gateway.StorageEventRequest;
+import co.worklytics.psoxy.gateway.StorageEventResponse;
+import co.worklytics.psoxy.storage.StorageHandler;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.events.S3Event;
+import com.amazonaws.services.lambda.runtime.events.models.s3.S3EventNotification;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.model.GetObjectRequest;
+import com.amazonaws.services.s3.model.ObjectMetadata;
+import com.amazonaws.services.s3.model.S3Object;
```

Comment:
these look version 1.x, not sure if that's gonna be deprecated.
I think this is the one we should be using. Just leave it for now, I don't know why we usethe 1.x of the lambdas either.
https://github.com/aws/aws-sdk-java-v2